calico-enterprise/getting-started/upgrading/upgrading-enterprise/kubernetes-upgrade-tsee/helm.mdx

```diff
@@ -74,22 +74,36 @@ These steps differ based on your cluster type. If you are unsure of your cluster
    kubectl apply --server-side --force-conflicts -f {{filesUrl}}/manifests/eck-operator-crds.yaml
    ```
 
-1. Run the Helm upgrade command for `tigera-operator`:
+1. Run the Helm upgrade command for `tigera-operator` and make sure to either update values.yaml with your configuration or use custom values.yaml file.
```

Comment:
I think I'm missing something. Why break up the step with two identical code blocks? It looks like the code is identical except for 'master', in which case the only change is the values.yaml path.

I would suggest that the  placeholder text can be the same, whether the values.yaml file is default or custom.

For example, `<path-to-values.yaml>`.

Reply:
I agree with Chris's statement. LIke this:

Run the Helm upgrade command for tigera-operator and update the values.yaml with your configuration, or use a custom values.yaml. For example:

helm upgrade calico-enterprise --values=<path-to-values.yaml> tigera-operator-v0.0.tgz \
--set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \
--namespace tigera-operator

Reply:
I followed the same style from helm docs [here](https://www.ibm.com/docs/en/b2b-integrator/6.1.2?topic=software-using-helm-command-line).
but yeah we can merge them, too. I have no strong opinion about it.